comments-on-conformance.md

```diff
@@ -1,16 +1,15 @@
-3.  WCAG 2 conformance is applied to the item being evaluated (i.e. web page) as a whole, except when a process includes use of several items, in which case all of the items that are needed in order to complete the process must conform.
-4.  In WCAG 2, when conformance relies on accessibility features of the platform (i.e. browser for web content) or on assistive technologies, WCAG 2 requires that there are assistive technologies, etc. that work with the product (web page). That is, conformance with WCAG 2 requires that the approaches used are supported by assistive technologies.
-5.  WCAG 2 allows information on part of a page to not conform if the same information is available elsewhere on the page in conforming fashion. However WCAG 2 identifies 4 success criteria that must be met on all areas of the page because they can interfere with the user's ability to access and use other parts of the page:
+1. The WCAG 2 success criteria, defined terms, and the [conformance requirements](https://www.w3.org/TR/WCAG22/#conformance) were designed to work together, such that the language of the success criteria is based on the nature of the conformance requirements and terminology.  The choice of what level to use for a given criteria (A vs. AA vs. AAA) was further influenced by a number of factors specific to the web domain, as set forth in [Understanding Levels of Conformance](http://www.w3.org/WAI/WCAG22/Understanding/conformance#levels).
+2. In the WCAG 2 conformance model, a [success criterion is met](https://www.w3.org/TR/WCAG22/#dfn-satisfies) if “the success criterion does not evaluate to ‘false’ when applied to the page”.  If the success criterion is about something that does not exist on the page being evaluated (e.g. a success criterion requiring captioning of audio, but there is no audio) the success criterion is automatically satisfied.  In such cases, some consider the criterion as “not applicable” but regardless the SC is satisfied.  This approach is central to the way the success criteria in WCAG are structured and worded.
+3.  WCAG 2 conformance is evaluated holistically, that is, for full pages and complete processes.
+4.  In WCAG 2, when conformance relies upon accessibility features of the platform (i.e. browser for web content) or on assistive technologies, WCAG 2 requires that there exists browsers and assistive technologies that work with the product (i.e., web page content).  That is, conformance with WCAG 2 requires that the approaches used are [supported by assistive technologies](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported).
+5.  WCAG 2 allows part of a page to not conform if the same information and functionality is available elsewhere on the page in a conforming manner.  However WCAG 2 identifies four success criteria that must be met on all areas of the page because they can interfere with the user’s ability to access and use other parts of the page:
```

Comment:
```suggestion
5.  WCAG 2 allows part of a page to not conform if the same information and functionality is available elsewhere on the page in a conforming manner.  However, WCAG 2 identifies four success criteria that must be met on all areas of the page because they can interfere with the user’s ability to access and use other parts of the page:
```